.gitignore

```diff
@@ -3,3 +3,4 @@ out
 .gradle
 build/
 output*/
+selfsigned.jks
```

Comment:
If the benchmarks are generating this in the runtime directory we shouldn't need to exclude this anymore.

Reply:
we renamed the file to make it more obvious that it's temporary, but it is still generated in the directory the jar/gradle is run from.